TLSharp.Core/Network/TcpTransport.cs

```diff
@@ -33,21 +39,19 @@ public async Task Send(byte[] packet)
 
             var tcpMessage = new TcpMessage(sendCounter, packet);
 
-            await _tcpClient.GetStream().WriteAsync(tcpMessage.Encode(), 0, tcpMessage.Encode().Length);
+            await _stream.WriteAsync(tcpMessage.Encode(), 0, tcpMessage.Encode().Length);
             sendCounter++;
         }
 
         public async Task<TcpMessage> Receieve()
```

Comment:
thanks for your contribution! now that I see this diff, I realise this typo above ^ (can you make a 2nd commit to rename this method to `Receive`? Thanks!